subgraphs/_reference_/src/sdk/protocols/lending/position.ts

```diff
@@ -303,6 +309,14 @@ export class PositionManager {
       .times(priceUSD);
     snapshot.blockNumber = event.block.number;
     snapshot.timestamp = event.block.timestamp;
+
+    snapshot.principal = this.position!.principal;
+    if (this.position!.side == PositionSide.BORROWER) {
```

Comment:
`if (this.market.borrowIndex && this.position!.side == PositionSide.BORROWER) { ... }`
same for supply index